src/main/java/pl/vishop/vishopplugin/config/Config.java

```diff
@@ -46,8 +54,42 @@ public boolean load(final ConfigurationSection cfg) {
             return false;
         }
 
+        this.broadcastMessages = new HashMap<>();
+        ConfigurationSection broadcastSection = cfg.getConfigurationSection("broadcasts");
+        boolean isSpigot = false;
+        try {
+            Class.forName("net.md_5.bungee.api.ChatColor");
+            isSpigot = true;
+        } catch (ClassNotFoundException ignored) {}
+
+        boolean translateHex = isSpigot && (Bukkit.getServer().getVersion().contains("1.16") || Bukkit.getServer().getVersion().contains("1.17") || Bukkit.getServer().getVersion().contains("1.18") || Bukkit.getServer().getVersion().contains("1.19"));
+        broadcastSection.getKeys(false).forEach(id -> {
+            List<String> messages = broadcastSection.getStringList(id);
+            int i = 0;
+            for (String msg : messages) {
+                msg = ChatColor.translateAlternateColorCodes('&', msg);
+                if (translateHex) msg = translateHex(msg);
+                messages.set(i, msg);
+                i++;
+            }
+
+            this.broadcastMessages.put(id, messages);
+        });
+
         this.taskInterval = 600L;
         return true;
     }
 
+    private static final Pattern hexPattern = Pattern.compile("&#[A-Fa-f0-9]{6}");;
+    private String translateHex(String message) {
+        Matcher matcher = hexPattern.matcher(message);
+        while (matcher.find()) {
+            String hex = message.substring(matcher.start(), matcher.end());
+            message = message.replace(hex, net.md_5.bungee.api.ChatColor.of(hex
```

Comment:
Tak nie chcemy robić, bo zepsujemy kompatybilność z Bukkitem - jeśli koniecznie chcesz użyć tego API, to trzeba z tego zrobić inną klasę, która może nie być w ogóle wołana